CheatSheetSetup.org

```diff
@@ -263,3 +264,5 @@
 # org-latex-inputenc-alist: (("utf8" . "utf8x"))
 # eval: (setq org-latex-default-packages-alist (cons '("mathletters" "ucs" nil) org-latex-default-packages-alist))
 # End:
+* test :ignore:
+# this is here to enable the inclusion of latex_headers before the beginning of the file.
```

Comment:
Oh that is interesting!

Good catch!

I did not notice this since I usually place all `latex_headers` in their own org-heading.

Reply:
happy to help :)